e2e-test/src/tests/cleanUp.test.js

```diff
@@ -0,0 +1,106 @@
+import {
+  vBtn,
+  vListTile,
+  vIcon,
+} from 'jest-puppeteer-vuetify';
+import { CLIENT_URL } from '../util';
+
+
+describe('relabel dataset', () => {
+  it('make dataset ready for analysis with 2 clicks', async () => {
+    await Promise.all([
+      page.goto(CLIENT_URL),
+      page.waitForNavigation({ waitUntil: 'networkidle0' }),
+    ]);
```

Comment:
Now that I'm looking at it, I think this step might be unnecessary. This snippet is also being run in a `beforeAll` in `jest.setup.js`, which should run it before every test. I'm curious if removing it breaks anything. I should have caught that sooner.